plugins/woocommerce/includes/wc-product-functions.php

```diff
@@ -9,6 +9,7 @@
  */
 
 use Automattic\Jetpack\Constants;
+use Automattic\WooCommerce\Enums\OrderStatus;
```

Comment:
Unused import: we should drop it before merging.

Reply:
Good catch, Vladimir! I have addressed all the unused import cases.